syslog.txt

```diff
@@ -0,0 +1,26 @@
+[info]tSep  5 18:36:50 rajat:systemd[1530]:: Starting Tracker metadata database store and lookup manager...
```

Comment:
@rajatchaple this one should also be added to .gitignore.

Reply:
removed from repo